openedx/core/djangoapps/auth_exchange/tests/test_views.py

```diff
@@ -181,7 +181,7 @@ def _verify_response(self, access_token, expected_status_code, expected_cookie_n
         Calls the login_with_access_token endpoint and verifies the response given the expected values.
         """
         url = reverse("login_with_access_token")
-        response = self.client.post(url, HTTP_AUTHORIZATION="Bearer {0}".format(access_token))
+        response = self.client.post(url, HTTP_AUTHORIZATION=u"Bearer {0}".format(access_token))
```

Comment:
I would argue this should actually be "b" prefixed since HTTP headers do not support unicode characters IIRC